rpcserver.go

```diff
@@ -2244,15 +2248,17 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 				linkActive = link.EligibleToForward()
 			}
 
-			isActive := peerOnline && linkActive
-			if !isActive {
+			if !linkActive {
 				continue
 			}
 
 			// Fetch the policies for each end of the channel.
 			chanID := channel.ShortChanID.ToUint64()
 			_, p1, p2, err := graph.FetchChannelEdgesByID(chanID)
 			if err != nil {
+				rpcsLog.Debugf("Unable to fetch the routing "+
```

Comment:
this should maybe be a fixup to a different commit? 😛

Reply:
Whoops, fixed.